egs2/chime2/enh1/local/clean_wsj0_data_prep.sh

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bash
```

Comment:
Is it possible to reuse the data prep in wsj recipe?

Reply:
No, this script is modified from the WSJ one and thus different. I just copied this script from Kaldi's recipe.